docs/src/concepts.md

```diff
@@ -0,0 +1,160 @@
+# Concepts
+
+## Multiphoton Calcium Imaging
+Over the past two decades, in vivo two-photon laser-scanning imaging of calcium signals has evolved into a mainstream modality for neurophysiology experiments to record population activity in intact neural circuits. The tools for signal acquisition and analysis continue to evolve but common patterns and elements of standardization have emerged.
```

Comment:
If you had time to hard wrap this, it will make future reviews easier. In sublime, it's `cmd+alt+q` to wrap to your ruler

Reply:
Sorry about that. I need to figure that out. Mac keyboard doesn't have the `alt` key.

Reply:
For mac, `alt`=`option`